Accent Change For Character Creation Page + Irish Accent

SsalamethVersaach opened the pull request:
<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
<!-- The text between the arrows are comments - they will not be visible on your PR. -->

## About the PR
<!-- What did you change in this PR? -->
This PR changed accents weights and also adds the ability to choose more accents in the character menu, as well as a new Irish accent. I did some yaml changes, along with adding a new accent.

## Why / Balance
<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
The reason behind this change was due to my own desire for people to have more accent choices, and to allow people to choose French, Russian, Spanish, Irish, and Mobster on round start. Part of this also stemmed from my desire to make it easier for players to diversify their characters.

## Technical details
<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
I changed the cost of some speech attributes in the speech.yml file, as well as added additional information and descriptions for these accents within the ftl files. In addition, I created an Irish accent which utilizes the irish.ftl and word_replacements.yml files- and added an accent system for it to function. 

## Media
<!-- 
PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
Small fixes/refactors are exempt.
Any media may be used in SS14 progress reports, with clear credit given.

If you're unsure whether your PR will require media, ask a maintainer.
-->
![screenshine](https://github.com/user-attachments/assets/fc0fe2e6-38fb-418b-b091-76b98c8fa957)

![image](https://github.com/user-attachments/assets/9605745c-0554-4d50-a00a-7444f163a019)

## Requirements
<!-- 
Due to influx of PR's we require to ensure that PR's are following the correct guidelines.


Please take a moment to read these if its your first time.

Check the boxes below to confirm that you have in fact seen these (put an X in the brackets, like [X]):
-->
- [X] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
<!--
List any breaking changes, including namespace, public class/method/field changes, prototype renames; and provide instructions for fixing them. This will be pasted in #codebase-changes.
-->
No breaking changes (that I know of).

**Changelog**
<!--
Make players aware of new features and changes that could affect how they play the game by adding a Changelog entry. Please read the Changelog guidelines located at: https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog
-->


<!--
Make sure to take this Changelog template out of the comment block in order for it to show up. Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog.-->
:cl:
- add: Added more accents on the character creation page, and the brand new Irish accent!

UBlueberry committed:
Guidebook Updates for the Amateur Spessman (#28603)

* Created NewPlayer.xml

* Created NewPlayer.yml and added it to guides.ftl

* shifted some controls from Space Station 14 to New? Start here!, as well as made a character creation xml to be written later

* switched some stuff between the New? entry and the Space Station 14 entry

* Made everything so nice!!!!!!!!!!!!!!

* fixed formatting inconsistencies

* added a How to use this guidebook section

* build correction and guidebook clarification for other servers

* wrote character creation ig probs fo shizzle

* added new terms to the glossary and alphabetized it

* meh this seems important enough to add

* okay no more shitsec bad idea

* I HATED Roleplaying.xml ANYWAY!!!

* I REALLY REALLY HATED IT ACTUALLYLLL

* Moved Controls and Radio into newplayer.yml, making meta.yml and radio.yml obsolete

* Separated the character creation bits that are just cosmetic from the ones that matter

* also put all the related new player xml files in their own folder

* expanded Radio.xml, kinda fixed survival.xml

* removed the line that mighta maybe sorta possibly could encourage self antag

* thought about this randomly but ICK OCK

* talking is no longer a key part of this game

* moves stuff around, a lot of stuff. basically moves everything but the jobs themselves around

* ah probably should make sure this works first also me when i lie on the internet

* don't be such a grammar nukie

Co-authored-by: Tayrtahn <tayrtahn@gmail.com>

* okay nevermind that's justified

Co-authored-by: Tayrtahn <tayrtahn@gmail.com>

* prepare. it is coming. the great reorganization...

* yesyes that's all well and goo- THE REORG IS COMING. THE REORG IS COMING. WAKE UP.

* rename that real quick

* step one begins. first, consolidate existing service entries into their own yaml. this makes botany.yml obsolete.

* update shiftandcrew.yml to only have departments as children also fuck it alphabetization

* consolidated salvage into cargo

* made a new entry for command

* gave salvage a home and service an existence

* made some XML files to be filled out later

* quick rename...

* took some stuff from Intro.txt and i think Gameplay.txt and put it in NewPlayer.xml

* The Great Writing about Departments (25XX, black and white)

* added a bunch of links everywhere

* biochemical is no longer a thing

* service formatinaaaaaaa

* shiny...,,,,,,,,, colo(u)rz..,,,,,,,,,,,,,

* let's get that fixed

* second time i made a typo there as well

* we hate fun around here

* grammar?!???/

* various fixes and more linkings

* oops

* wewlad lol!!

---------

Co-authored-by: Tayrtahn <tayrtahn@gmail.com>
Co-authored-by: AJCM <AJCM@tutanota.com>

EmoGarbage404 committed:
add default page support for the guidebook (#28772)

AJCM-git committed:
Open the guidebook when people with less than an hour playing join the round (#28774)

* Open the guidebook when people with less than an hour playing join the round

* Filter for gameplayState

* Fix tests

* tweaks

* saltern update (#28773)

Co-authored-by: deltanedas <@deltanedas:kde.org>

* Fix admin menu objects list (#28787)

* Make `MakeAntag()` log errors instead of throwing exceptions (#28771)

Make `MakeAntag()` log errors instead of throw

* add default page support for the guidebook (#28772)

* Probably a better way to handle this

---------

Co-authored-by: deltanedas <39013340+deltanedas@users.noreply.github.com>
Co-authored-by: Leon Friedrich <60421075+ElectroJr@users.noreply.github.com>
Co-authored-by: Nemanja <98561806+EmoGarbage404@users.noreply.github.com>

EmoGarbage404 committed:
Redo new player & controls guidebooks (#28794)

EmoGarbage404 committed:
Mining Rebalance (#30920)

* first pass

* this shit too

* ok fix that shit

* buff

* actually fix that

SsalamethVersaach committed:
Commit for accent changes - Rus, Mobster, Fr, Sp

SsalamethVersaach committed:
Commit for accent changes - Rus, Mobster, Fr, Sp

SsalamethVersaach committed:
Commit for accent changes - Rus, Mobster, Fr, Sp

SsalamethVersaach committed:
Did the accent changes - Rs, Sp, Fr, Mobster

SsalamethVersaach committed:
Did the accent changes - Rs, Sp, Fr, Mobster

SsalamethVersaach committed:
Did the accent changes - Rs, Sp, Fr, Mobster

SsalamethVersaach committed:
Irish Accent now recognized.

SsalamethVersaach committed:
Irish Accent now recognized.

SsalamethVersaach committed:
Irish Accent now recognized.

SsalamethVersaach committed:
Fixed some accent weights

SsalamethVersaach committed:
Fixed some accent weights

SsalamethVersaach committed:
Fixed some accent weights

SsalamethVersaach committed:
Changed accent weights again -Russian w/o stutter

SsalamethVersaach committed:
Changed accent weights again -Russian w/o stutter

SsalamethVersaach committed:
Changed accent weights again -Russian w/o stutter

SsalamethVersaach committed:
It stopped working

SsalamethVersaach committed:
It stopped working

SsalamethVersaach committed:
It stopped working

SsalamethVersaach committed:
Maybe this fixes it again?

SsalamethVersaach committed:
Maybe this fixes it again?

SsalamethVersaach committed:
Maybe this fixes it again?

SsalamethVersaach committed:
Fixed everything, it seems to work! - Neutral.yml

SsalamethVersaach committed:
Fixed everything, it seems to work! - Neutral.yml

SsalamethVersaach committed:
Fixed everything, it seems to work! - Neutral.yml

SsalamethVersaach committed:
Merge branch 'DeltaV-Station:master' into funny-accent-change

SsalamethVersaach committed:
Changed the weight of Russian

SsalamethVersaach committed:
Changed the weight of Russian

SsalamethVersaach committed:
Changed the weight of Russian

SsalamethVersaach committed:
Merge branch 'funny-accent-change' of https://github.com/SsalamethVersaach/Delta-v into funny-accent-change

SsalamethVersaach committed:
Removed Irish ref

SsalamethVersaach committed:
Removed Irish ref

SsalamethVersaach committed:
Removed Irish ref

DangerRevolution commented on RobustToolbox:
Bad

SsalamethVersaach replied:
I don't know what to do about this as it's a submodule and automatically populated in the PR.

DangerRevolution commented on Resources/Prototypes/Traits/speech.yml:
This needs to be in a Delta-V folder

DangerRevolution commented on Resources/Prototypes/DeltaV/Traits/neutral.yml:
Why?

SsalamethVersaach replied:
This was where I was going to put the new traits, just was testing there prior and forgot about it.

DangerRevolution commented on Resources/Prototypes/Traits/speech.yml:
```diff
@@ -64,14 +64,12 @@
   description: trait-socialanxiety-desc
   category: SpeechTraits
-  cost: 2
+  cost: 1
```
This is unrelated to your PR, and should be a commented change

SsalamethVersaach replied:
It was related due to me wanting more variety, but comments have been added-

DangerRevolution commented on Resources/Prototypes/Traits/speech.yml:
```diff
@@ -84,6 +82,51 @@
   description: trait-frontal-lisp-desc
   category: SpeechTraits
-  cost: 2
+  cost: 1
```
This is unrelated to your PR, and should be a commented change

SsalamethVersaach replied:
It was related due to me wanting more variety, but comments have been added (to all of them)

DangerRevolution commented on Resources/Locale/en-US/traits/traits.ftl:
This needs to be in a Delta-V folder.

Russian accent is really bad and leads to extremely poor readability. Should not be used.

The Descriptions are also just copy and pasted with minor edits; should be unique.

SsalamethVersaach replied:
Russian Accent was removed, traits were moved to a Delta-V folder, and I changed descriptions.

DangerRevolution reviewed (changes requested):
Russian accent is up for debate but stuff needs to be commented / in delta-V folders

SsalamethVersaach committed:
Merge branch 'DeltaV-Station:master' into funny-accent-change

SsalamethVersaach committed:
Re-updating

SsalamethVersaach committed:
Re-updating

SsalamethVersaach committed:
Re-updating

SsalamethVersaach committed:
Merge branch 'funny-accent-change' of https://github.com/SsalamethVersaach/Delta-v into funny-accent-change

themias committed:
Add trash tag to chopsticks (#31260)

PJB3005 committed:
Word wrap for vote titles (#31248)

EmoGarbage404 committed:
fix and buff the handheld mass scanner (#31284)

PJBot committed:
Automatic changelog update

beck-thompson committed:
Fix nukies not getting station codes (#31272)

First commit

PJBot committed:
Automatic changelog update

Brandon-Huu committed:
tweak(shuttle_infiltrator): Replace NukeCodePaper with BoxFolderNuclearCodes (#31273)

PJBot committed:
Automatic changelog update

PJBot committed:
Automatic changelog update

Dutch-VanDerLinde committed:
fix vox/dionae/arachnids/moths/slimes not having burn visuals (#31286)

inital

Sarahon committed:
Long human ears (#30490)

Long human ears colour fix. Shortcut removed.

PJBot committed:
Automatic changelog update

DrSmugleaf committed:
Update RobustToolbox to v230.1.0 (#31297)

Winkarst-cpu committed:
Make vending machine use EntityPrototypeView (#30064)

* Make vendor machines use EntityPrototypeView

* Update

* 1

* Kill me

* For the love of god!!!

PJBot committed:
Automatic changelog update

EmoGarbage404 committed:
Small strange jackboots buff (#30586)

* Small strange jackboots buff

* Update ClothingSlowOnDamageModifierComponent.cs

PJBot committed:
Automatic changelog update

Commit:
Add Vox atmosphere marker, vents, and air alarms

Commit:
Make component fields datafields

metalgearsloth committed:
Predict IntrinsicUI (#31310)

metalgearsloth committed:
Fix inventory GUI for non-inventory entities (#31306)

Default to invisible and only set visible if itemslots gets initialised.

Siigiil committed:
Fixes the projectile list in AmmoShotEvent when firing something with a ProjectileSpreadComponent (#31300)

Fixed multiple projectile addition.

PJBot committed:
Automatic changelog update

Mervill committed:
Remove some wayward folder includes in the `csproj` (#31244)

remove some wayward folder inlcudes

EmoGarbage404 committed:
Move mining roid and magnet pulls closer (#31296)

move mining roid and magnet pulls closer

PJBot committed:
Automatic changelog update

lzk228 committed:
No meta implanters + other (#31045)

* No meta implanters + other

* Update implanters.yml

PJBot committed:
Automatic changelog update

Mervill committed:
Remove obsolete functions from tile commands. (#31112)

* first part

* second part - file scope namespaces

* missing peice

lzk228 committed:
Give clown the second jester suit (#30673)

* Give clown the second jester suit

* hat

PJBot committed:
Automatic changelog update

TheShuEd committed:
fix Gatherable magic number (#31327)

Update GatherableSystem.cs

metalgearsloth committed:
Update submodule to 230.2.0 (#31338)

PJBot committed:
Automatic changelog update

dffdff2423 committed:
Make Vox blood able to be used to make space cleaner (#30749)

Boiling vox blood creates ammonia

PJBot committed:
Automatic changelog update

Sarahon committed:
Emote sprites rework and update. (#30887)

Emoji eye change

PJBot committed:
Automatic changelog update

jimmy12or committed:
Recipe for cream (#30503)

* Adds heat-based recipe for cream

* Added aeration

---------

Co-authored-by: Alchebun <jimmy.12.or#gmail.com>

metalgearsloth committed:
Station AI overlay (#31335)

Split

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Fix AI vision occlusion (#31341)

Forgot to do this, but how tf do doors work then.

SlamBamActionman committed:
Add codeword highlighting (#30092)

* Added codeword highlighting

* Updated to support more codeword roles, color is set serverside

* Review feedback

* Change to a Component-based system using SessionSpecific

* Tidied up CanGetState, set Access restrictions on component

* Clean-up

* Makes the injection ignore brackets, restore some codewords, remove "Taste/Touch" from adjectives

PJBot committed:
Automatic changelog update

lzk228 committed:
fix metal pitcher name (#31351)

SlamBamActionman committed:
Add Galoshes slowdown over slippery surfaces (#30967)

* first draft

* Fixed it all, just need to rename stuff

* Rename and add comments

* Clean-up

* Access added

PJBot committed:
Automatic changelog update

lzk228 committed:
very smol emergency boxes cleanup (#31355)

lzk228 committed:
cleanup bakery weapons (#31356)

Commit:
Increased Chest Marking Points for Reptilian Race (#30786)

changed reptile chest marking count to 3

PJBot committed:
Automatic changelog update

FATFSAAM2 committed:
new boxer figurine voicelines (#31382)

initial comnit

PJBot committed:
Automatic changelog update

Winkarst-cpu committed:
Fix the folded chair damage inspection (#31378)

* Fix the folded folding chair

* Whoops

PJBot committed:
Automatic changelog update

slarticodefast committed:
Make storage implant closable using the action again (#31381)

EmoGarbage404 committed:
add new salv loot into the vgroid procgen (#31290)

add new loot into vgroid procgen

PJBot committed:
Automatic changelog update

EmoGarbage404 committed:
Add space debris to salvage magnet (#31113)

* add space debris to salvage magnet

* multiplication

* necessary mercy for NPCs

* back and forth on mercy

PJBot committed:
Automatic changelog update

Magicalus committed:
Stop nanotrasen sign from rotating (#31388)

make snapCardinals false for the nanotrasen sign.

Co-authored-by: Zachary Yona <magicalusf@gmail.com>

Winkarst-cpu committed:
Make holding down an arrow key traverse the pointer in chat (#31380)

* Make holding down an arrow key traverse the pointer in chat

* Better way

PJBot committed:
Automatic changelog update

erohrs2 committed:
ID Change to Dinnerware Vending Machine (#31225)

Dinnerware Vending Machine access changed from Service to Kitchen

Co-authored-by: Preston Smith <92108534+thetolbean@users.noreply.github.com>

PJBot committed:
Automatic changelog update

Commit:
Line 44, story-generation.ftl: "chrisp" is spelled as "crisp" (#31401)

chrisp is spelled as crisp

PJB3005 committed:
"LV" is an acronym (#31402)

DevilishMilk committed:
Moths can now eat all bandanas (#31405)

* Update bandanas.yml

* Update bandanas.yml

* Re-add ClothMade to botany bandana

PJBot committed:
Automatic changelog update

PursuitInAshes committed:
Change `Resave` Permission Requirement to Host (#31397)

ChangePermissionFlagToHost

PJBot committed:
Automatic changelog update

eoineoineoin committed:
Remove client state from server AirAlarmComponent (#31236)

* Remove client state from server AirAlarmComponent

Send information for all connected devices, not just the ones for the
current tab, as attempting to limit this breaks multiple users viewing
the same UI.

Fixes #12842

* Send device data as a list, rather than a dictionary

---------

Co-authored-by: Eoin Mcloughlin <helloworld@eoinrul.es>

PJB3005 committed:
Fix ChemVend jugs again (#31398)

This re-introduces the dummy entity naming code, originally introduced in #29178 and randomly removed by #30064 with no technical justification given.

Fixes #31373

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Fix dungeon spawn + ftl overlap (#31413)

* Fix dungeon spawn + ftl overlap

* Better fixes

PJBot committed:
Automatic changelog update

themias committed:
Fix firelockedge rotation (#31371)

save

PJBot committed:
Automatic changelog update

PJBot committed:
Automatic changelog update

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Verb tweaks (#31309)

* Verb tweaks

Remove the LOS check because this is already done above in CanExamine.

* Fix outlines

* import

metalgearsloth committed:
Add ContainerComp (#31311)

Applies EntProtoId changes upon insertion / removal from container. Can also be useful for borgs / mechs / vehicles in future but atm I just used it for AI.

EmoGarbage404 committed:
Blueprints (#31138)

* Blueprints

* Update tables_loot.yml

* doink

* mark as required

PJBot committed:
Automatic changelog update

deltanedas committed:
decouple ItemToggle from PowerCellDraw (#31392)

* remove ItemToggle from PowerCellDraw query

* add EntityQuery for resolves, make them all optional

* move integration to ToggleCellDraw

* add ToggleCellDraw to almost every PowerCellDraw prototype

* :trollface:

* :trollface:

* :trollface:

* let it disable on mapinit

* set update time on mapinit, make borg power logic consistent now

* :trollface:

---------

Co-authored-by: deltanedas <@deltanedas:kde.org>

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Power stuff (#31314)

* Power stuff

- Add shared IsPowered
- Add shared ResolveApc
- Move PowerChangedEvent to shared for now
- Add SlimPoweredLight that actually functions how you'd expect a PoweredLight to function it id didn't have a bunch of bloat on it.

* big update

* boing

metalgearsloth committed:
ItemToggle + slots stuff (#31312)

* ItemToggle + slots stuff

- Add component for itemslot locks to match LockComponent (surprised this didn't exist).
- Add thing for pointlight to match itemtoggle. In future should be used for PDAs and stuff but need to fix some other stuff first.

* Also this

* grill

metalgearsloth committed:
Run actions on press (#31191)

I know UI generallys want to do stuff on key-up not key-down. With that being said the delay on release for these is noticeable.
e.g.
ActivateInWorld / UseInHand / any in-world interactions are on-press. This is especially noticeable if you have the same event on action + verb.

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Action stuff (#31305)

* Action stuff

- Cleanup some event stuff
- Avoid dirtying entity unnecessarily
- Add ActionGrant as an easy way to apply / remove actions via compregistry.

* Fix merge

SsalamethVersaach committed:
Merge branch 'DeltaV-Station:master' into funny-accent-change

metalgearsloth committed:
Update submodule to 231.0.0 (#31430)

SsalamethVersaach committed:
Some minor changes with comments

SsalamethVersaach committed:
Some minor changes with comments

SsalamethVersaach committed:
Some minor changes with comments

SsalamethVersaach committed:
Merge branch 'funny-accent-change' of https://github.com/SsalamethVersaach/Delta-v into funny-accent-change

SsalamethVersaach committed:
Update ScottishAccentComponent.cs (Why did it change)

I don't know why this said I changed it

Signed-off-by: SsalamethVersaach <thebluettdle@gmail.com>

SsalamethVersaach committed:
Update ScottishAccentComponent.cs (Why did it change)

I don't know why this said I changed it

Signed-off-by: SsalamethVersaach <thebluettdle@gmail.com>

SsalamethVersaach committed:
Merge branch 'funny-accent-change' of https://github.com/SsalamethVersaach/Delta-v into funny-accent-change

SsalamethVersaach committed:
Please fix the ScottishAccentComponent being a-

SsalamethVersaach committed:
Please fix the ScottishAccentComponent being a-

SsalamethVersaach reviewed (commented):
I think I fixed everything (hopefully).

Please let me know what else is needed. 

Thanks.

TheShuEd committed:
Weather fix (#30857)

* weather fix

* localize errors

metalgearsloth committed:
ContainerComp fix (#31434)

Forgot to push these ones on the branch.

metalgearsloth committed:
togglelight updates (#31435)

Missed in the green checkmark sea

PJB3005 committed:
Increase (SUPER)PACMAN ramp rate/tolerance (#31403)

All PACMANs used to use the same ramping parameters, this meant a SUPERPACMAN (50 kW) took literally a 100 seconds to get up to its full output level. Ouch.

PACMAN has been raised to 5000 W/s with 1500 W tolerance, SUPERPACMAN to 7500 W/s with 2500 W tolerance

PJBot committed:
Automatic changelog update

Blackern5000 committed:
Make space scanning T1 technology (#31321)

make space scanning and mass scanner t1 tech

PJBot committed:
Automatic changelog update

ShadowCommander committed:
Add a bool to disable buckling on InteractHand (#30001)

PJBot committed:
Automatic changelog update

Brandon-Huu committed:
fix(FieldGenerators): Don't show nonsense popup (#30469)

* .

MilonPL committed:
fix justice id cards (#1683)

trolled fuck idk

lzk228 committed:
Moff cooking. Part one. (#30668)

* yml part

* guidebook

* commit

* some fixes

* a little

* change

PJBot committed:
Automatic changelog update

metalgearsloth committed:
Predict EmitSoundOnUIOpen (#31464)

* Predict EmitSoundOnUIOpen

BUI prediction led to this moment.

* unforky

Moomoobeef committed:
Bots now make sound when they speak (#31471)

fixes 31454

PJBot committed:
Automatic changelog update

Winkarst-cpu committed:
Change the science radio channel color (#31319)

* Change the science radio channel color

* This one looks better

PJBot committed:
Automatic changelog update

Winkarst-cpu committed:
Move OnGhostAttempt to GhostSystem (#31445)

* Move OnGhostAttempt to GhostSystem

* Remove unused dependencies and sort them

slarticodefast committed:
Fix energy sword visuals (#31478)

PJBot committed:
Automatic changelog update

Mervill committed:
Replace obsolete code in shuttle systems. (#31408)

* Format DockingSystem.Shuttle

* arrivals system

* docking system

* shuttle console system

* emergency shuttle system

* shuttle system

* thruster system

* Fix compile error

---------

Co-authored-by: Pieter-Jan Briers <pieterjan.briers+git@gmail.com>

QueerNB committed:
Replaces small fans on most shuttles with directional fans. (#31495)

Mervill committed:
Replace obsolete functions in NPC systems (#31448)

12rabbits committed:
Make guidebook remember where you left off (#31375)

* Guidebook tracks last read entry

* Persistent guidebook fixes

* Persistent guidebook requested changes

* Update Content.Client/UserInterface/Systems/Guidebook/GuidebookUIController.cs

---------

Co-authored-by: Pieter-Jan Briers <pieterjan.briers@gmail.com>

PJBot committed:
Automatic changelog update

Commit:
Revert "add atmos stuff for mapping vox boxes (#1668)"

This reverts commit 6dc5ff7720bcadaaa570f466ffc2f936ac010125.

IamVelcroboy committed:
Upates spawners with entity tables (#1661)

* Update spawners using entity tables

* fixes

* Remove space

* Make rare actually rare

* Comment out Space Law. Let anarchy prevail!

---------

Co-authored-by: Velcroboy <velcroboy333@hotmail.com>

Commit:
:trollface:

Timemaster99 committed:
Cherry-pick Harpy Updates (#1641)

* Harpy Peacock Tail (#428)

Courtesy of @StillIcarus

This adds a new Peacock tail marking to Harpies. I'm also taking this
time to update them to the recent DeltaV changes to the birbs.

![image](https://github.com/Simple-Station/Einstein-Engines/assets/16548818/4f776006-700e-4d92-802c-03e6dd201491)

![image](https://github.com/Simple-Station/Einstein-Engines/assets/16548818/5fd143ab-069f-4d1c-b5e0-ee982a95b4b6)

The backside is semi cursed, but that's something I can't do within the
scope of this PR due to sprite system limitations that Death has planned
improvements for. Just ignore them. :)

Have a birb as consolation.

![faridaiscute](https://github.com/Simple-Station/Einstein-Engines/assets/16548818/09373a57-f32f-48eb-b52b-6d12143f62ff)

:cl: VMSolidus
- add: Peacock Tails have been added for Harpies

---------

Signed-off-by: VMSolidus <evilexecutive@gmail.com>
Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>

* Birbs Have Wings Too (#477)

# Description

During PR #464, somehow Moth's got back their ability to move with their
wings whenever the gravity gets shut off. Well, I'm here to correct the
injustice by also putting it on Harpies. In fact, Harpies have a much
bigger claim to it, their wings are absolutely massive compared to
Moths.

# Changelog

:cl:
- add: Harpies remembered that just like moths; they also have wings,
and can zoom whenever the gravity turns off.

Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>

* Harpy Sprite Update 3 (#556)

This will be my third sprite update pack for Harpies, with new sprites
courtesy of StillxIcarus (Discord). These include owl wings, and forked
tails. I've also personally made a new "Large Ear Tufts" variation. Many
sprites have been split between Huescale and Whitescale variations, with
Huescale offering greater variety of colors, while Whitescale offers
better pure whites(A notable weakness of having only huescale options
was that white winged harpies ended up a yellow-cream color).

Additionally, I have updated all of the Harpy Hardsuit and Softsuit
variations to now include **Wing Sleeves**, with some hardsuits even
having personalized color schemes for their sleeves, depending on how
important the wearer may be. Of course, all of the syndicate hardsuits
have received their own unique schemes for each suit, including for the
first time a species-specific Elite Syndicate Suit.

![image](https://github.com/user-attachments/assets/f4e90e0d-6869-47de-bd26-a6668f8e3e2b)

![image](https://github.com/user-attachments/assets/ecc98a25-936c-46a4-bddc-ff3a25b27b25)

Additionally, I am taking this as a convenient time to move a majority
of the Harpy files to the core EE Directory, as I am taking direct
responsibility for maintaining my creation here, rather than anywhere
else.

:cl:
- add: Harpy Update 3, New markings for Harpies, all new
species-specific hardsuit & softsuit variations that feature wing
sleeves.

* White Dream Harpy Markings (#696)

ONIKS was kind enough to provide me with several new Harpy markings,
consisting of alternate Bat-wings, Haven Tail, Long Forked Tail, and
Swallow Tail. I have also taken the liberty of actually making a basic
set of "Bionic Wings", which are just a slightly edited version of the
"harpsuit" wing sleeve, including hardsuit lights. Unfortunately due to
limitations in the game's code concerning markings, I couldn't make the
lights use the unshaded shader. This limitation irritates me. I'll
probably fix that in a separate PR.

I actually also received a pack of Arachnid and Felinid sprites! But I'm
going to add those in a different PR entirely.

<details><summary><h1>Media</h1></summary>
<p>

Bionic wings

![image](https://github.com/user-attachments/assets/6acc64a9-be6b-435b-9c61-91287ef65c10)

Bat Wings

![image](https://github.com/user-attachments/assets/e44fe12e-8410-47ca-a4c1-5e644fd36b8a)

Haven tail

![image](https://github.com/user-attachments/assets/2adadcdd-04fc-458d-8491-61bd52a3b4d1)

Swallow tail

![image](https://github.com/user-attachments/assets/615391eb-0314-4e6d-a87b-101d0a307955)

Long Forked Tail

![image](https://github.com/user-attachments/assets/e3d672f7-b8cb-4497-a2a3-87d0c224847d)

</p>
</details>

:cl:
- add: The following new markings have been added for Harpies: Bat
Wings, Simple Bionic Wings, Haven Tail, Swallow tail, and Long Forked
Tail

* Split files

* Harpy Hotfix (Cherry-Pick Broke Things) (#570)

Captain's Carapace inherited HidesHarpyWings because wizden made it
parent off Large outerwear for reasons. I've moved HidesHarpyWings to
only LargeOuterwear that actually have harpy sprites so that this won't
happen again.

![image](https://github.com/user-attachments/assets/84ee06e3-eeeb-4193-8cb0-492bd1d3d82e)

![image](https://github.com/user-attachments/assets/ac9493b3-75e7-4696-b531-18b00d1332a5)

* Uncommented digitigrade inv template

* Removed duplicate locale entry

* Add comment

---------

Signed-off-by: VMSolidus <evilexecutive@gmail.com>
Co-authored-by: VMSolidus <evilexecutive@gmail.com>
Co-authored-by: Danger Revolution! <142105406+DangerRevolution@users.noreply.github.com>

DeltaV-Bot committed:
Automatic changelog update

Commit:
Cherry-Pick Oni Markings (#1691)

* Oni Horn Markings (#726)

# Description

---

Three new Oni horn markings,] two of which have three tone support.
There was a distinct lack of large horns, which I tried my hand at.

<details><summary><h1>Media</h1></summary>
<p>


![display](https://github.com/user-attachments/assets/cb0604e8-941c-48fd-8fb8-205893b87c5c)

</p>
</details>

---

# Changelog

<!--
You can add an author after the `:cl:` to change the name that appears
in the changelog (ex: `:cl: Death`)
Leaving it blank will default to your GitHub display name
This includes all available types for the changelog
-->

:cl:
- add: Added crowned, tall bull, and tall curved Oni horns.

* Oni Pointy Ears And Tusks (#727)

# Description

Adds eight pointy ears, two tusks, one feet marking (toes resembling
WarCraft trolls) and human nose markings to Onis. The pointed ears are a
big reference to tieflings (similar to Onis) who often have them.

Onis will now have pointy ears by default, but there is a marking to
disable ears.

5 human nose markings have all been made available to Onis, Harpies, and
Felinids. Onis and Felinids can now get chest scars, originally
humna-only. The tusk markings are originally from Frontier goblins.

The new pointy ears give players the power to make their Onis look like
**tieflings, orcs, elves, goblins**, and more fantasy races.

These markings can potentially be used in the future for other species
like psionic elves.

## Media

**Blood Elf** (Tall Ears)


![image](https://github.com/user-attachments/assets/257c8839-303c-45b2-b129-3f9183e47695)

<details><summary>See more</summary>

**Orc** (Upward Ears, Large Tusks)


![image](https://github.com/user-attachments/assets/46bac694-de88-4e04-a12b-7ff8a8111e7b)

**Night Elf** (Wide Ears)


![image](https://github.com/user-attachments/assets/18813660-cd2c-4d37-800d-bf84b1d7dee5)

**Night Elf** (Slanted Ears)


![image](https://github.com/user-attachments/assets/7f39457b-875d-4c46-a5fb-428a6a535fb1)

**Troll** (Large Ears, Small Tusks, Two-Toed Feet)


![image](https://github.com/user-attachments/assets/d9a0c52e-020d-4b55-bb7e-7a4afed61dcc)

**Character Setup**


![image](https://github.com/user-attachments/assets/35ba27d4-e6d1-4c52-bbc7-0f057df12fc8)


![image](https://github.com/user-attachments/assets/5d9e7690-7118-48b3-95b9-50ee784660e2)

</details>

## Changelog

:cl: Skubman
- add: Onis have received eight new pointy ear markings in the 'Head
(Side)' section that can help them resemble tieflings, orcs, elves,
goblins, trolls, and more.
- add: Onis get two new tusk markings in the 'Snout' section, and a
Two-Toed feet marking in the 'Legs' section.
- add: Felinids, Onis, and Harpies can now select Human nose markings
like the schnozz, blobby nose, and nubby nose.
- add: Felinids and Onis can now select the Chest Scar and Top Surgery
Scar markings.
- tweak: Default Onis now use the Double Curved horn and Upward Ears
markings.

* [S] Onis: Get A Boatload Of Troll Horns. (#741)

# Description

Adds **16** new horns to Onis, 10 of which are inspired by **Homestuck**
trolls. Most of the new horns have three-tone variants, and some also
have striped variants.

New horns and available variants:
- Antlers (Three Tone)
- Aries (Three Tone)
- Double Thick (Striped, Three Tone)
- Erebia (Ringed, Three Tone)
- Inclined (Three Tone)
- Makara (Striped, Three Tone)
- Nepeta (Three Tone)
- Pisces (Striped, Three Tone)
- Sagittarius
- Serket (Three Tone)
- Taurus (Striped, Three Tone)
- Tavris (Three Tone)
- Unicorn
- Vantas (Three Tone)
- Virgo (Three Tone)
- Wavy (Striped, Three Tone)
- Three Tone for the existing Double Curved horn
- Three Tone for the existing Double Curved Outwards horn

## Media

**Serket (Three Tone)**


![image](https://github.com/user-attachments/assets/69beabb3-01ea-49ac-9e9e-8d9351bf99df)

<details><summary>See more</summary>

**Makara (Striped)**


![image](https://github.com/user-attachments/assets/839707b7-c063-41f4-b59f-dc36f2126967)

**Aries (Three Tone)**


![image](https://github.com/user-attachments/assets/ba7a65da-06d0-4ddd-b0d0-afa9cff5ad03)


![image](https://github.com/user-attachments/assets/b4abcdd2-a11e-4d4f-be61-1970e78cf588)

**Antlers**


![image](https://github.com/user-attachments/assets/cfb47d5b-f985-4dbe-b86d-96c31472007f)

**Pisces (Striped)**


![image](https://github.com/user-attachments/assets/9426c51e-49fe-41d4-b049-ee8b1a159c57)

**Wavy**


![image](https://github.com/user-attachments/assets/c29420aa-4576-49ec-af08-0ff38aff7ea4)

**Taurus (Striped)**


![image](https://github.com/user-attachments/assets/9aa3084e-4acc-473e-b875-465b242d0551)

**Nepeta (Three Tone)**


![image](https://github.com/user-attachments/assets/639abc4a-0cf1-456a-b214-6520e39e992f)

**Sagittarius (Three Tone)**


![image](https://github.com/user-attachments/assets/2a4a7c6f-0678-462d-8de6-f8fe6c10aef3)

**Unicorn**


![image](https://github.com/user-attachments/assets/6783cbd4-2dcf-4204-bb03-3f3601e1c6f7)


![image](https://github.com/user-attachments/assets/c09f7ef3-a0e3-429f-be70-76db0232ad01)

</details>

## Changelog

:cl: Skubman
- add: 16 new horns for Onis have been added, with three-tone variants
and some striped variants! The new horns include the Serket, Nepeta,
Vantas, Makara, and more.
- add: The Oni horns Double Curved and Double Curved Outwards have
received three-tone variants.

* Respacing to EE

* I forgot human.yml, apparently, but we're fiiine.

---------

Co-authored-by: dootythefrooty <137359445+dootythefrooty@users.noreply.github.com>
Co-authored-by: Angelo Fallaria <ba.fallaria@gmail.com>

DeltaV-Bot committed:
Automatic changelog update

deltanedas committed:
job whitelists panel !!! (#1678)

* make JobPrototype.Whitelisted respect blanket whitelist, make jobs use it

* add jobwhitelists command and ui

* add job whitelist button to player panel

---------

Co-authored-by: deltanedas <@deltanedas:kde.org>

DeltaV-Bot committed:
Automatic changelog update

Commit:
some fixing

Commit:
rename deltav implanters

Commit:
merge master

Commit:
uncomment Drinks from guide help now that it exists

deltanedas committed:
Merge pull request #1693 from deltanedas/upstream-ops

upstream merge 27/8

DeltaV-Bot committed:
Automatic changelog update

deltanedas committed:
fix nyano coat loadouts (#1697)

Co-authored-by: deltanedas <@deltanedas:kde.org>

DeltaV-Bot committed:
Automatic changelog update

deltanedas committed:
more helix price (#1696)

Co-authored-by: deltanedas <@deltanedas:kde.org>

SsalamethVersaach committed:
Checking if this fixes merge issues

SsalamethVersaach committed:
Checking if this fixes merge issues

SsalamethVersaach committed:
Just trying to fix this stupid thing

SsalamethVersaach commented:
Apparently GitHub has decided to PR literally EVERYTHING but my changes in this so I am just going to re-PR it, I give up on comprehending it. Apologies.
